### PR TITLE
Externalize js: base_maps.html

### PR DESCRIPTION
**Pull request by orangejenny**
Ready for review; don't merge yet so I can be sure to be around post-deploy and test custom reports.

@millerdev / @kaapstorm / @gcapalbo

### CODE REVIEW

File: corehq/apps/reports/static/reports/js/base_maps.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+hqDefine("reports/js/base_maps.js", function() {
+    var initial_page_data = hqImport("hqwebapp/js/initial_page_data.js").get;
+    CONTEXT = initial_page_data('context');
```
Comment (millerdev) [inline corehq/apps/reports/static/reports/js/base_maps.js]: Can we define these locally inside this `hqDefine` block, or are they accessed globally? If global (yuck!), would be nice to mark the globals at the top of the file.

File: custom/ewsghana/static/ewsghana/js/map.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+hqDefine("ewsghana/js/map.js", function() {
+    var initial_page_data = hqImport("hqwebapp/js/initial_page_data.js").get;
+    CONTEXT = initial_page_data('context');
```
Comment (millerdev) [inline custom/ewsghana/static/ewsghana/js/map.js]: same as other comment on local vs global.

File: custom/opm/static/opm/js/map.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+hqDefine("opm/js/map.js", function() {
+    var initial_page_data = hqImport("hqwebapp/js/initial_page_data.js").get;
+    CONTEXT = initial_page_data('context');
```
Comment (millerdev) [inline custom/opm/static/opm/js/map.js]: same as other comment on local vs global.

**Review by millerdev — APPROVED** [review]
Would love to see those undefined variables become locals, but understand if out of scope for this PR.

**Review by gcapalbo — APPROVED** [review]

**Comment by orangejenny** [comment]
@millerdev Reduced the usage of globals. `ICON_PATH` is used in `config.dataTables.bootstrap.js` which I'm not inclined to fiddle with in this PR, so I just moved those assignments out of the function definitions.

**Review by millerdev — APPROVED** [review]

**Pull request merged**